golang/trace/zipin_tracereporter_test.go

```diff
@@ -44,15 +44,14 @@ func TestZipkinTraceReporterFactory(t *testing.T) {
 }
 
 func TestBuildZipkinSpan(t *testing.T) {
-	host := &tchannel.Endpoint{
-		Ipv4:        "127.0.0.1",
-		Port:        8888,
+	endpoint := tchannel.TargetEndpoint{
+		HostPort:    "127.0.0.1:8888",
 		ServiceName: "test",
```

Comment:
use different values for `ServiceName` and `Name` so that you know you're not accidentally swapping values around.